portfolio/src/main/webapp/blogScript.js

```diff
@@ -0,0 +1,37 @@
+window.addEventListener('DOMContentLoaded', (event) => {
```

Comment:
This part seems to be a repeat of script.js - consider also including script.js in your blog page, and only putting the parts not-in-common here?